Cargo.toml

```
@@ -8,25 +8,32 @@ description = "Async modem api for the nRF9160"
 homepage = "https://github.com/diondokter/nrf-modem"
 repository = "https://github.com/diondokter/nrf-modem"
 readme = "README.md"
-keywords = ["nRF9160", "LTE", "GPS", "NB-IoT", "embedded"]
+keywords = ["nRF9160", "nRF9151", "LTE", "GPS", "NB-IoT", "embedded"]
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-nrfxlib-sys = "2.4.2"
+nrfxlib-sys = "2.7.0"
```

Please make this pinned. I should have done that with the previous version already.
```suggestion
nrfxlib-sys = "=2.7.0"
```